common/alignment/alignment-en.html

```diff
@@ -0,0 +1,98 @@
+---
+{
+	"title": "GCWeb Alignment example",
+	"language": "en",
+	"altLangPage": "alignment-fr.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
```

Comment:
Just remove it, like on the French page.

```suggestion
```

Note: Eventually we should simply apply the same breadcrumb for the whole GCWeb site.